db/seeds.rb

```diff
@@ -2,7 +2,7 @@
 
 if Rails.env.development?
   token = AuthToken.create(user: user, user_agent: 'Rails')
-  token.update(hashed_secret: BCrypt::Password.create('secret', cost: 1), device_id: 'device-id')
+  token.update(device_id: 'device-id')
```

Comment:
This could just be `AuthToken.create(user:, user_agent: 'Rails', device_id: 'device-id')` not. I think it was just separated because of the `before_validation` in AuthToken that generated the secret. (Which wasn't even a problem now that I look at it.)